Lib/glyphsLib/builder/guidelines.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import (print_function, division, absolute_import,
+                        unicode_literals)
+
+from glyphsLib.types import point
```

Comment:
`point` is imported but unused

Reply:
I will remove it.

It's here because it was used in the WIP implementation of `to_glyphs_guidelines` that I replaced with `pass`. Is it still the plan to have this MR merged quickly with stub functions for everything UFO -> Glyphs?

Reply:
> Is it still the plan to have this MR merged quickly with stub functions for everything UFO -> Glyphs?

yes!